### PR TITLE
Release Note update for June 26 Hot Fix for Magento 2.3.2 and 2.3.1

### PR DESCRIPTION
**Pull request by jfrontain**
## Purpose of this pull request

This pull request adds information to the Magento 2.3.2 and 2.3.1 release notes about the June 26, 2019 hot fix for the async/bulk API scope issue. 

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.3/release-notes/ReleaseNotes2.3.2OpenSource.html
https://devdocs.magento.com/guides/v2.3/release-notes/ReleaseNotes2.3.2Commerce.html

https://devdocs.magento.com/guides/v2.3/release-notes/ReleaseNotes2.3.1OpenSource.html
https://devdocs.magento.com/guides/v2.3/release-notes/ReleaseNotes2.3.1Commerce.html
- ...
- ...

## Links to Magento source code

- ...
- ...

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=4853) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=4853) before we can accept your contribution.<br/><hr/>**Jeanne Frontain** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

File: guides/v2.3/release-notes/ReleaseNotes2.3.1Commerce.md [hunk 1]
```diff
@@ -4,14 +4,18 @@ title: Magento Commerce 2.3.1 Release Notes
...
 
+## Apply the Scope parameter for Async/Bulk API patch to address an issue with the Async/Bulk REST API
 
-## Apply patch PRODSECBUG-2198 to address critical SQL injection vulnerability
+In certain versions of Magento Open Source and Magento Commerce, the Asynchronous and Bulk REST endpoints support the default store view scope only. After this patch is applied to deployments running those versions of Magento, the current Magento message queue implementation 
+will factor in the store that executes queue operations. See [Patch for Magento Framework Message Queue and Store Scopes](https://community.magento.com/t5/Magento-DevBlog/Patch-for-Magento-Framework-Message-Queue-and-Store-Scopes/ba-p/135209) for a full discussion of this scope-related issue and patch contents. See [Applying patches](https://devdocs.magento.com/guides/v2.3/comp-mgr/patching.html) for specific instructions on downloading and applying Magento patches. Navigate to the [Magento Security Center](https://magento.com/security/patches), and select the patch associated with the version of Magento you are running.
```
Comment (meker12) [inline guides/v2.3/release-notes/ReleaseNotes2.3.1Commerce.md]:
```suggestion
will factor in the store that executes queue operations. See [Patch for Magento Framework Message Queue and Store Scopes](https://community.magento.com/t5/Magento-DevBlog/Patch-for-Magento-Framework-Message-Queue-and-Store-Scopes/ba-p/135209) for a full discussion of this scope-related issue and patch contents. See [Applying patches](https://devdocs.magento.com/guides/v2.3/comp-mgr/patching.html) for specific instructions on downloading and applying Magento patches. To apply the patch, navigate to the [Magento Security Center](https://magento.com/security/patches), and select the patch associated with the version of Magento you are running. 
```

File: guides/v2.3/release-notes/ReleaseNotes2.3.1OpenSource.md [hunk 2]
```diff
@@ -4,15 +4,19 @@ title: Magento Open Source 2.3.1 Release Notes
...
 
+## Apply the Scope parameter for Async/Bulk API patch to address an issue with the Async/Bulk REST API
 
-## Apply patch PRODSECBUG-2198 to address critical SQL injection vulnerability
+In certain versions of Magento Open Source and Magento Commerce, the Asynchronous and Bulk REST endpoints support the default store view scope only. After this patch is applied to deployments running those versions of Magento, the current Magento message queue implementation 
+will factor in the store that executes queue operations. See [Patch for Magento Framework Message Queue and Store Scopes](https://community.magento.com/t5/Magento-DevBlog/Patch-for-Magento-Framework-Message-Queue-and-Store-Scopes/ba-p/135209) for a full discussion of this scope-related issue and patch contents. See [Applying patches](https://devdocs.magento.com/guides/v2.3/comp-mgr/patching.html) for specific instructions on downloading and applying Magento patches. Navigate to the [Magento Security Center](https://magento.com/security/patches), and select the patch associated with the version of Magento you are running.
```
Comment (meker12) [inline guides/v2.3/release-notes/ReleaseNotes2.3.1OpenSource.md]:
```suggestion
will factor in the store that executes queue operations. See [Patch for Magento Framework Message Queue and Store Scopes](https://community.magento.com/t5/Magento-DevBlog/Patch-for-Magento-Framework-Message-Queue-and-Store-Scopes/ba-p/135209) for a full discussion of this scope-related issue and patch contents. See [Applying patches](https://devdocs.magento.com/guides/v2.3/comp-mgr/patching.html) for specific instructions on downloading and applying Magento patches. To apply the patch, navigate to the [Magento Security Center](https://magento.com/security/patches), and select the patch associated with the version of Magento you are running. 
```

File: guides/v2.3/release-notes/ReleaseNotes2.3.2OpenSource.md [hunk 3]
```diff
@@ -3,13 +3,19 @@ group: release-notes
...
+ 
+
+## Apply the Scope parameter for Async/Bulk API patch to address an issue with the Async/Bulk REST API
+
+In certain versions of Magento Open Source and Magento Commerce, the Asynchronous and Bulk REST endpoints support the default store view scope only. After this patch is applied to deployments running those versions of Magento, the current Magento message queue implementation 
+will factor in the store that executes queue operations. See [Patch for Magento Framework Message Queue and Store Scopes](https://community.magento.com/t5/Magento-DevBlog/Patch-for-Magento-Framework-Message-Queue-and-Store-Scopes/ba-p/135209) for a full discussion of this scope-related issue and patch contents. See [Applying patches](https://devdocs.magento.com/guides/v2.3/comp-mgr/patching.html) for specific instructions on downloading and applying Magento patches. Navigate to the [Magento Security Center](https://magento.com/security/patches), and select the patch associated with the version of Magento you are running.
```
Comment (meker12) [inline guides/v2.3/release-notes/ReleaseNotes2.3.2OpenSource.md]:
```suggestion
will factor in the store that executes queue operations. See [Patch for Magento Framework Message Queue and Store Scopes](https://community.magento.com/t5/Magento-DevBlog/Patch-for-Magento-Framework-Message-Queue-and-Store-Scopes/ba-p/135209) for a full discussion of this scope-related issue and patch contents. See [Applying patches](https://devdocs.magento.com/guides/v2.3/comp-mgr/patching.html) for specific instructions on downloading and applying Magento patches. To apply the patch, navigate to the [Magento Security Center](https://magento.com/security/patches), and select the patch associated with the version of Magento you are running. 
```

File: guides/v2.3/release-notes/ReleaseNotes2.3.2OpenSource.md [hunk 4]
```diff
@@ -62,48 +68,6 @@ Magento now performs the following tasks as **asynchronous background processes*
...
-* **GraphQL caching**. GraphQL can now cache the `category`, `cmsBlocks`, `cmsPage`, `products`, and `urlResolver` queries, which improves response times.  To enable this feature, send these queries using HTTP GET. You can send all other queries with HTTP GET, but they are not cached.
-
-* **GraphQL performance test scenario coverage**. We have added PWA GraphQL test scenarios for critical checkout and catalog browsing to the performance builds. GraphQL community developers can use the new scenarios to measure storefront performance. <!--- MC-15826, 15922-->
-
-See [Release notes](https://devdocs.magento.com/guides/v2.3/graphql/release-notes.html) for a more detailed discussion of recent GraphQL bug fixes.
-
-### Progressive Web Apps (PWA)
-
-* **Improved modular component library**. PWA Studio continues to build out the concept for functional and data components through the Peregrine library. Components can now be reused and scaled for frontend needs. Magento has planned a phased rollout for Peregrine functional and data components, starting with the Search component which is launching with this release.
-
```
Comment (meker12) [inline guides/v2.3/release-notes/ReleaseNotes2.3.2OpenSource.md]: Looks like the PWA, GraphQL, and Inventory Management highlights were deleted.

**Comment by jfrontain** [comment]
running tests

**Comment by jfrontain** [comment]
running tests

**Pull request merged**